tests/foreman/api/test_convert2rhel.py

```diff
@@ -299,7 +296,6 @@ def test_convert2rhel_oracle_with_pre_conversion_template_check(
             'inputs': {
                 'Activation Key': activation_key_rhel.id,
                 'Restart': 'yes',
```

Comment:
This change is part of an ongoing feature for Convert2Rhel improvement so I would suggest to attach jira link here as well with `Verifies` docstring.